microsoft/PowerToys
# [ScreenRuler] Use Strings/*/Resources.resw instead of Properties/ one

## Summary of the Pull Request


## PR Checklist

- [x] **Closes:** #27521


## Detailed Description of the Pull Request / Additional comments


## Validation Steps Performed
Run PT and ScreenRuler
Check that tooltips are there and show proper text
Use the accessibility Insights tool to check AutomationProperties.Name property
